src/ape_ethereum/__init__.py

```diff
@@ -15,6 +15,11 @@ def converters():
     yield int, WeiConversions
 
 
+@plugins.register(plugins.ConversionPlugin)
+def time_converters():
+    yield str, TimestampConversions
+
+
```

Comment:
So, for the easy conversion things, where accessing config manager, network manager, or other conversions (via conversion manager) wasn't necessary, I "cheated" a bit and defined them directly in the conversion manager's file:
https://github.com/ApeWorX/ape/blob/main/src/ape/managers/converters.py#L92

Everything is set None there because they didn't link all of their dependencies like a normal conversion should do

Reply:
Also, note for future knowledge, you can only register all of your converters 1 time, so the way to add it here (for other type of converters where dependencies are needed, or where it makes more sense for it to live in the plugin, like with `WeiConversions`) is to add another yield line to `converters` above, not duplicate the function twice e.g.:

```diff
  @plugins.register(plugins.ConversionPlugin)
  def converters():
      yield int, WeiConversions
+     yield str, TimestampConversions
```

Reply:
I was stuck between this option and what I did. I reverted to this now